tools/replicate_raisinbread.pl

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/perl -w
+
+=pod
+
+=head1 NAME
+
+replicate_raisinbread.pl -- Create a local copy of the RaisinBread dataset and
+store each file as a symlink to the original dataset found in /data-raisinbread. 
+
+=head1 SYNOPSIS
+
+perl replicate_raisinbread.pl C<[/path/to/raisinbread]> C<[/path/to/output_dir]> 
+
+=head1 DESCRIPTION
+
+This script takes in two arguments. The first argument is the path to the
+RaisinBread dataset (typically C</data-raisinbread>) and the path to the
+directory where the replicated dataset will be stored. For example, suppose
+the script is run with the following arguments:
+
+perl replicate_raisinbread.pl C</data-raisinbread> C</data>
+
+The replicated dataset will be found in C</data/data-raisinbread/> 
+
+=head1 LICENSING
+
+License: GPLv3
+
+=head1 AUTHORS
+
+LORIS community <loris.info@mcin.ca> and McGill Centre for Integrative Neuroscience
+
+=cut
+
+use strict;
+use warnings;
+
+use Cwd qw/abs_path/;
+use File::Find;
+use File::Spec::Functions qw/catfile abs2rel/;
+use File::Path qw/make_path/;
+
+my $Usage = <<USAGE;
+usage:  $0 /path/to/raisinbread /path/to/output_dir
+
+This script creates a local copy of the RaisinBread dataset and stores each file 
+as a symlink to the original dataset.
+
+This script takes in two arguments. The first argument is the path to the
+RaisinBread dataset (typically /data-raisinbread) and the path to the
+directory where the replicated dataset will be stored. For example, suppose
+the script is run with the following arguments:
+
+perl replicate_raisinbread.pl /data-raisinbread /data/data-raisinbread/
+
+The replicated dataset will be found in /data/data-raisinbread/
+
+USAGE
+
+# Checks if the right number of arguments are given
+if (scalar(@ARGV) != 2) {
+    print "ERROR: Incorrect number of arguments\n";
+    print $Usage;
+    exit 1;
```

Comment:
For exit codes, you should import the library in `uploadNeuroDB/NeuroDB` called `ExitCode.pm` at the top of the script and use one of the preexisting exit code listed in https://github.com/aces/Loris-MRI/blob/main/uploadNeuroDB/NeuroDB/ExitCodes.pm

In this particular case, it would be `$NeuroDB::ExitCodes::MISSING_ARG`

(example of such usage: https://github.com/aces/Loris-MRI/blob/f9df1b78cd96882009264d7fba122b294c7c6329/uploadNeuroDB/imaging_upload_file.pl#L168)

Reply:
I added these exit codes to the script and I also renamed it to `replicate_raisinbread_for_mcin_dev_vm.pl`.